scripts/DeployFraxOFTProtocol/DeployFraxOFTProtocol.s.sol

```diff
@@ -428,6 +432,42 @@ contract DeployFraxOFTProtocol is BaseL0Script {
         });
     }
 
+    function setMovementEnforcedOptions(
+        address[] memory _connectedOfts
+    ) public virtual {
+        bytes memory optionsTypeOne = OptionsBuilder.newOptions().addExecutorLzReceiveOption(5_000, 0);
+        bytes memory optionsTypeTwo = OptionsBuilder.newOptions().addExecutorLzReceiveOption(5_000, 0);
+
+        L0Config[] memory configs = new L0Config[](1);
+        configs[0] = nonEvmConfigs[1]; // mapped to movement
+
+        setEnforcedOptions({
+            _connectedOfts: _connectedOfts,
+            _configs: configs,
+            _optionsTypeOne: optionsTypeOne,
+            _optionsTypeTwo: optionsTypeTwo
+        });
+    }
+
+    function setAptosEnforcedOptions(
+        address[] memory _connectedOfts
+    ) public virtual {
+        // TODO: change these, [200_000, 2_500_000] is solana config
+        bytes memory optionsTypeOne = OptionsBuilder.newOptions().addExecutorLzReceiveOption(5_000, 0);
+        bytes memory optionsTypeTwo = OptionsBuilder.newOptions().addExecutorLzReceiveOption(5_000, 0);
+
+        L0Config[] memory configs = new L0Config[](1);
+        configs[0] = nonEvmConfigs[1]; // mapped to Aptos
```

Comment:
Should be `nonEvmConfigs[2]`

Reply:
I managed to get aptos & movement on same addresses